src/test/java/LoginTests.java

```diff
@@ -17,7 +17,7 @@ public void LoginEmptyEmailPasswordTest() {
         WebDriver driver = new ChromeDriver(options);
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example

For example `public void loginEmptyEmailPasswordTest ()`